### DISCUSSION TITLE
Quarkus application to Commonhaus!

### DISCUSSION THREAD

**Discussion by dandreadis** [commonhaus/foundation#184] [Joining Commonhaus]
### Project information

- **Project name**: Quarkus
- **Project website**: https://quarkus.io/
- **Code repository**: https://github.com/quarkusio/quarkus
- **License**: Apache 2.0

### Do you have authority to represent this project?

- [X] I own or maintain this project

### Describe your project

Quarkus is a Kubernetes Native Java stack tailored for OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards.

On behalf of the Quarkus team, for reasons explained in the [Quarkus Blog](https://quarkus.io/blog/quarkus-in-a-foundation/), and after a [consultation period](https://github.com/quarkusio/quarkus/discussions/41108), we concluded that it will be beneficial for the Quarkus project and the greater Quarkus community, for [Quarkus to join the Commonhaus Foundation](https://quarkus.io/blog/quarkus-moving-to-commonhaus/).

Let the fun begin :)

### Things that must be done (bootstrap phase)

- [X] **Review the bylaws**, especially sections about [projects](https://www.commonhaus.org/bylaws/cf-membership.html#projects), [decision making](https://www.commonhaus.org/bylaws/decision-making.html), and [voting for significant decisions](https://www.commonhaus.org/bylaws/cf-council.html#voting).
- [X] **Read the [bootstrapping agreement](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md)**.
- [X] If all is well, and you agree to participate in the bootstrapping phase to help us build this thing, then **create a PR** with the following:

    1. A filled out copy of the bootstrapping agreement. You can make a copy of [bootstrapping.md](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md) file and check that in, or checkin a filled out [pdf](https://github.com/commonhaus/foundation/releases/download/final-draft/bootstrapping.pdf). Please follow naming conventions.
    2. An updated [CONTACTS.yaml](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml) that names your project representative in the EGC section
    3. An updated [PROJECTS.yaml](https://github.com/commonhaus/foundation/blob/main/PROJECTS.yaml) that provides details / describes your project
    4. An updated [TRADEMARKS.md](https://github.com/commonhaus/foundation/blob/main/TRADEMARKS.md) that adds your project name and project logo (if you have one). Please include the footnote reference, as we are using your project name and logo with your permission until we have a formal agreement in place and resources are transferred.

    The commit should be "Signed-off-by" your GH login (`git commit -s -m "<project> bootstrapping agreement, yyyy-mm-dd"`)

**Comment by dandreadis** [comment]
[Adding Quarkus for bootstrap](https://github.com/commonhaus/foundation/pull/183/files)

**Comment by gavinking** [comment]
Excellent!

**Comment by ebullient** [comment]
Your application has been approved. Welcome to Commonhaus! 🎉

We have a lot to talk about.

Next steps for @dandreadis: 

- accept Commonhaus organization invitation
- visit https://www.commonhaus.org/member/ to review required materials and set up email alias.
